test/qa/test_meshnet_other.py

```diff
@@ -99,3 +99,28 @@ def test_set_defaults_when_logged_out_1st_set(tech, proto, obfuscated):
     assert "Settings were successfully restored to defaults." in sh.nordvpn.set.defaults()
 
     assert settings.app_has_defaults_settings()
+
+
+# This doesn't directly test meshnet, but it uses it
+def test_set_post_quantum_on_meshnet_enabled():
+
+    pq_alias = settings.get_pq_alias()
+
+    with pytest.raises(sh.ErrorReturnCode_1) as ex:
+        sh.nordvpn.set(pq_alias, "on")
```

Comment:
Here variable is not needed - use function immediately

Reply:
fixed